From d43dbdca09431b2c6e49fc255dc5c714d3056574 Mon Sep 17 00:00:00 2001
From: Matthias Schiffer <mschiffer@universe-factory.net>
Date: Sat, 15 Jun 2019 23:30:23 +0200
Subject: [PATCH] ar71xx: Onion Omega is BROKEN

This device doesn't have an Ethernet port, and even with the Ethernet
shield, eth0 is not configured.

(cherry picked from commit a44a5dce02de3738944b7ee742e5bc2fa068c939)
---
 docs/index.rst         | 4 ----
 targets/ar71xx-generic | 4 +++-
 2 files changed, 3 insertions(+), 5 deletions(-)

diff --git a/docs/index.rst b/docs/index.rst
index 31e8e977..bbb3296a 100644
--- a/docs/index.rst
+++ b/docs/index.rst
@@ -174,10 +174,6 @@ ar71xx-generic
 
   - Koala [#ath10k]_
 
-* Onion
-
-  - Omega
-
 * OpenMesh
 
   - A40
diff --git a/targets/ar71xx-generic b/targets/ar71xx-generic
index bd028dfa..9f3f8f77 100644
--- a/targets/ar71xx-generic
+++ b/targets/ar71xx-generic
@@ -137,7 +137,9 @@ factory
 
 # Onion
 
-device onion-omega onion-omega
+if [ "$BROKEN" ]; then
+device onion-omega onion-omega # BROKEN: no ethernet
+fi
 
 
 # OpenMesh
-- 
2.17.1

